Update Contributing.md (#75)
letsgamedev/Suffragium: CONTRIBUTING.md

## Merge pull request conditions

> Necessary conditions:
> - No checks are failing
> - at least one upvote, that isn't from the creator of the pr
> - 75% positive votes
> 
> If **additionally** one of the following conditions is true, the PR will me merged.
> - **Last commit is 24h or older** and **has more than 10 votes**
> - **Last commit is 72h or older**

Reviewer: ```suggestion
When **all conditions** are met, the PR will be merged.
- All checks have passed
- 75% positive votes (Creator vote does not count)
- Last commit is older than 24h and has more than 10 votes **or** Last commit is older than 72h
```
